src/cct3.cc

```diff
@@ -389,9 +391,33 @@ SharedWavefunction psi4_cct3(SharedWavefunction ref_wfn, Options& options)
             onebody,
             twobody,
             rep_e,
-            scf_e
+            scf_e,
+            corl_e,
+            corl_e_sec
             );
 
+    std::string eneprim, enesec;
+    if (calc_type == 1) {
+        enesec = "CCSD";
+        eneprim = "CCSD";
+    } else if (calc_type == 2) {
+        enesec = "CCSD";
+        eneprim = "CR-CC(2,3)";
+    } else if (calc_type == 3) {
+        enesec = "CCSDt";
+        eneprim = "CCSDt";
+    } else if (calc_type == 4) {
+        enesec = "CCSDt";
+        eneprim = "CC(t;3)";
+    }
+
+    ref_wfn->set_module("cct3");
+    ref_wfn->set_scalar_variable("CURRENT CORRELATION ENERGY", corl_e);
+    ref_wfn->set_energy(scf_e + corl_e);
```

Comment:
Also, `set_energy` doesn't seem to be passing the right energy to python. I get the reference energy back when I test the H8-0.1 example.

Reply:
huh, the `compare_values` is failing in H8 test? if you add a line here `ref_wfn->set_scalar_variable("CURRENT ENERGY", scf_e + corl_e);` does that help? `wfn.energy_` and `wfn.variables_["CURRENT ENERGY"]` are auto-synched in current psi (past year+) but mayn't be if your psi is older.

Reply:
That solved it. I'm running PSI4 1.3.2 from conda-forge.